Readme.md

```diff
@@ -41,6 +41,22 @@ node index.js PATH_TO_UD-Viz 8000
 
 Installing [Visual Studio Code](https://code.visualstudio.com/) is recommended, in order to use the plugin formatter [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). Once installed you should setup Prettier with single quote coding style (Preferences => Settings => Type in search bar : Single quote => Toggle single quote of Prettier extension)
 
+### Documentation
+
+The documentation can be found [here](./docs/html/index.html)
+
+It is created using [JSDOC](https://jsdoc.app/) and [Clean-Jsdoc-Theme](https://github.com/ankitskvmdam/clean-jsdoc-theme) 
+
+To generate the documentation : 
+
+```bash
+git clone https://github.com/VCityTeam/UD-Viz.git
+cd UD-Viz
+npm install
+npm run docs
+```
+
+
```

Comment:
Ok the mardown sources seem to have been moved into docs/static/Doc/...
What is this static distinction for ? Is there a dynamic part ? Or is the distinction between source = static and target ? Or maybe you are using static for mardown type of source ?!
Independently from the static part, do you need the "Doc" repetition (this is already in docs, right?).